api/python/provisioner/commands/__init__.py

```diff
@@ -514,7 +585,18 @@ def dynamic_validation(self, params: inputs.SWUpdateRepo, targets: str):
         )
 
         # TODO IMPROVE VALIDATION EOS-14350
-        #   - there is no other candidate that is being verified
+        #   - there is no other candidate that is being verified:
+        #     if found makes sense to raise an error in case the other
+        #     logic is still running, if not - forcibly remove the previous
+        #     candidate
+        #   - after first mount 'sw_update_candidate' listed in disabled repos
+        if self._does_repo_exist(f'sw_update_{candidate_repo.release}'):
+            logger.warning(
+                'other repo candidate was found, proceed with force removal'
```

Comment:
@dmitrydb could you update the message: `proceeding`

Reply:
Fixed